<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(39, 40, 47), stop:1 rgb(63, 65, 76));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(39, 40, 47, 255), stop:1 rgb(63, 65, 76));</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>36</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/UI/images/logo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="head">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>105</y>
     <width>72</width>
     <height>72</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>32</horstretch>
     <verstretch>32</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/UI/images/headimg-login.png);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nameLb">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>215</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(251, 251, 251);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="Line" name="lineup">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>272</y>
     <width>265</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 50, 60);
background-color: rgb(48, 50, 60);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>245</y>
     <width>265</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(71, 75, 94);
background-color: transparent;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdEdit">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>330</y>
     <width>265</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(71, 75, 94);
background-color: transparent;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="linedown">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>360</y>
     <width>265</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 50, 60);
background-color: rgb(48, 50, 60);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="passwdLb">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>300</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(251, 251, 251);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoLoginCheck">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>390</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(146, 148, 168);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="remberPwd">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>390</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(146, 148, 168);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="forget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(92, 180, 20);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>忘记您的密码？</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="logoButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>265</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(251, 251, 251);
	border:0px groove gray;
    border-radius:5px;
    padding:2px 4px;
    border-style: outset;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(61,163,241, 255), stop:1 rgba(36,95,207, 255));
}
QPushButton:hover{
	color: rgb(251, 251, 251);
	border:0px groove gray;border-radius:5px;padding:2px 4px;border-style: outset;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(36, 95, 207, 255), stop:1 rgba(61, 163, 241, 255));
}
QPushButton:pressed{
	color: rgb(251, 251, 251);
	border:0px groove gray;border-radius:5px;padding:2px 4px;border-style: outset;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(61,163,241, 255), stop:1 rgba(36,95,207, 255));
}</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
